src/hotspot/share/code/relocInfo.cpp

```diff
@@ -35,6 +35,8 @@
 #include "runtime/stubCodeGenerator.hpp"
 #include "utilities/align.hpp"
 #include "utilities/copy.hpp"
+#include <new>
+#include <type_traits>
```

Comment:
Please add an empty blank line between HotSpot includes and system includes. We don't explicitly state this, but this was the style that we generated when IncludeDB was removed.

Reply:
Done.

Reply:
Thanks.